models/orderDao.js

```diff
@@ -0,0 +1,168 @@
+        error.statusCode = 500;
+        throw error
+    }
+};
+// 장바구니 상품 삭감
+const cutProduct = async(verifiedEmail,productId) => {
+    try{
+        const userId = await appDataSource.query(`
+        select id from users where email = '${verifiedEmail}'
+        `)
+        const extractedId = userId[0].id
+        const product_id = await appDataSource.query(`
+        select id from products where id = '${productId}'
+        `)
+        const extProduct_id = product_id[0].id
+        await appDataSource.query(`
+        update basket
+        set basket.quantity = basket.quantity - 1
+        where basket.user_id = '${extractedId}' and basket.product_id = '${extProduct_id}'
+        `)
+    
+    }catch(err){
+        console.log(err)
+        middleErr.error(500 , 'INVALID_DATA_INPUT' )
+    }
+};
+
+//장바구니 상품 삭제
+const deleteProduct = async(verifiedId, productId)=>{
+    try{
+        const userId = await appDataSource.query(`
+        select id from users where id = '${verifiedId}'
+        `)
+        const extId = userId[0].id
+        const product_id = await appDataSource.query(`
+        select id from products where id = '${productId}'
+        `)
+        const extProductId = product_id[0].id
+        return await appDataSource.query(`
+        DELETE FROM basket
+        WHERE user_id = '${extId}' AND product_id = '${extProductId}';
+        `)
+    }catch(err){
+        console.log(err)
+        middleErr.error(500, 'INVALID_DATA_INPUT')
+    }
+};
+
+// 장바구니 담기
+const addBusket = async(verifiedUserId, product_id)=>{
+    try{
+        return await appDataSource.query(`
+        insert into basket(user_id, product_id, quantity, status_id)
+        values (?,?,?,?)
+        `, [verifiedUserId, product_id, 1, 1])
+        
+    }catch(err){
+        console.log(err)
+        middleErr.error(500, 'INVALID_DATA_INPUT')
+    }
+};
+
+
+
+//오더 테이블 생성
+ const payBusket = async(verifiedId, order_number,address, products, total_price)=>{
+    try{
+        for (let i=0; i<products.length; i++){
+            console.log(products[i].id, products[i].quantity)
+            const product_id = products[i].id
+            const product_quantity = products[i].quantity
+        await appDataSource.query(`
+        insert into orders (order_number, user_id, address, product_id, product_quantity, total_price)
+        values (?,?,?,?,?,?)
+        `,[order_number, verifiedId, address, product_id, product_quantity, total_price ]);
+        } 
+        
+        await appDataSource.query(`
+        update basket
+        set basket.status_id = basket.status_id + 1
+        where basket.user_id = ? and basket.status_id = ?`,[verifiedId , 1])
+
+
+    }catch(err){
+        console.log(err)
+        middleErr.error(500, 'INVALID_DATA_INPUT')
+    }
+ }
+
+//잔여포인트 확인
+const remainingPoints = async(verifiedId)=>{
+        return await appDataSource.query(`
+        select point
+        from users
+        where id = ?
+        `, [verifiedId])
+}
+
+
```

Comment:
```suggestion
//잔여포인트 확인
const getPointsByUserId = async(userId) => {
  return await appDataSource.query(`
    SELECT point
    FROM users
    WHERE id = ?
  `, [userId])
}


```
- 함수는 동사로 시작하는 이름으로 명명하여 어떤 역할을 하는지 명확히 작성하시는 것이 좋습니다